### PR TITLE
Combine same-module and alphabetize imports

### PR DESCRIPTION
**Pull request by sirreal**
There were multiple imports from the following paths, which is a lint error and easy source of bugs and confusion:

- `state/sites/plans/selectors`
- `lib/plans`

Combine same-path imports (fixes lint error).
Alphabetize imports (🤓).

No functional changes.

Lint error spotted in #23802 

## Testing
- Smoke test some plans pages.

### CODE REVIEW

File: client/my-sites/plan-features/index.jsx [hunk 1]
```diff
@@ -3,56 +3,63 @@
...
-import { getPlanPath, canUpgradeToPlan, applyTestFiltersToPlansList } from 'lib/plans';
+import {
+	getPlanDiscountedRawPrice,
+	getPlansBySiteId,
+	isCurrentUserCurrentPlanOwner,
+} from 'state/sites/plans/selectors';
+import {
+	getPlanPath,
+	canUpgradeToPlan,
+	applyTestFiltersToPlansList,
```
Comment (tyxla) [inline client/my-sites/plan-features/index.jsx]: 🔤 here as well?

File: client/my-sites/plan-features/index.jsx [hunk 2]
```diff
@@ -3,56 +3,63 @@
 /**
  * External dependencies
  */
-
+import classNames from 'classnames';
+import page from 'page';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
```
[the remark this answers is not in the document]
Comment (sirreal) [reply client/my-sites/plan-features/index.jsx]: Lately I've been wondering what value that adds and what the impact is. It's a special case –generally something to discourage– and makes automated tooling incompatible. Let's just get em sorted.

File: client/my-sites/plan-features/index.jsx (hunk 1 above)
Comment (sirreal) [reply client/my-sites/plan-features/index.jsx]: HOW DID I MISS THIS?!?!?! 😭

File: client/my-sites/plan-features/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (tyxla) [reply client/my-sites/plan-features/index.jsx]: 🔤 allthethings!

**Comment by sirreal** [comment]
More alphabetized and more rebased 💪 

I'll give it a spin and merge.

**Pull request merged**